cmd/pj-rehearse-metrics/pj-rehearse-metrics.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"cloud.google.com/go/storage"
+	"context"
+	"flag"
+	"fmt"
+	"google.golang.org/api/iterator"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"sync"
+)
+
+var (
+	originCiBucket   = "origin-ci-test"
+	rehearseJobName  = "pull-ci-openshift-release-master-pj-rehearse"
+	metricsPathInRun = "artifacts/rehearse-metrics.json"
+)
+
+type scraper struct {
+	cacheDir string
+	bucket   *storage.BucketHandle
+	ctx      context.Context
+}
+
+func (s *scraper) scrapeRun(runPrefix string, prNumber string) {
+	runNumber := path.Base(runPrefix)
+	metricsPath := path.Join(runPrefix, metricsPathInRun)
+
+	prDir := filepath.Join(s.cacheDir, prNumber)
+	localMetricsPath := filepath.Join(prDir, runNumber)
+	if _, err := os.Stat(localMetricsPath); err == nil {
+		return
+	}
+
+	reader, err := s.bucket.Object(metricsPath).NewReader(s.ctx)
+	if err != nil {
+		return
+	}
+	data, err := ioutil.ReadAll(reader)
+	_ = reader.Close()
```

Comment:
is this a typo?

Reply:
Not really, GoLand suggested this as an explicit error ignore, so I used it. I'll probably just defer it.